Library/Homebrew/brew.rb

```diff
@@ -88,7 +88,7 @@ def require?(path)
   end
 
   if internal_cmd
-    Homebrew.send cmd.to_s.tr("-", "_").downcase
+    Homebrew::InstallCommand.send cmd.to_s.tr("-", "_").downcase
```

Comment:
Hi @alyssais! Thats a good approach, but unfortunately it does not work in our case because ```InstallCommand``` is defined in only some of the ```command``` files. For example, it is not defined in the tests.rb, so when we run ```brew tests```, it gives the same error ```uninitialized constant Homebrew::InstallCommand```.